app/mixins/routes/disallow-authenticated.js

```diff
@@ -0,0 +1,17 @@
+import Ember from "ember";
+
+export default Ember.Mixin.create({
+  requireAuthentication: false,
+  beforeModel: function(){
+    return new Ember.RSVP.Promise((resolve, reject) => {
+      if (this.session.get('isAuthenticated')) {
+        resolve();
+      } else {
+        this.session.fetch('aptible').then(resolve, reject);
+      }
+    }).then(() => {
+      this.transitionTo('index');
+    }, function(){});
```

Comment:
The idea that rejections are no-ops but resolutions, which indicate that the user is authenticated, get redirected to `/`? I would have expected a no-op successful `resolve` when the user is in the desired state (not authenticated), and a `.catch` at the end of the promise that transitions the authenticated user to '/'.